.github/workflows/ci.yml

```diff
@@ -35,6 +35,8 @@ env:
   DB_RESET: "true"
   VERBOSE: "true"
   DOCKER_CACHE: "pulled"
+  # Do not show all output lines on CI. Only output of the failed tests will be shown as a summary
+  TEST_SHOW_ALL_OUTPUT_LINES_AS_THEY_APPEAR: "false"
```

Comment:
Do we want this configurable at all? If we are running a single test (i.e. in sequential mode) then we still want output to show up, and if we are running parallel tests then we can't have this and force it off anyway.

Reply:
I thought about it and was not sure either. I simply find it rather unnecessary to print all the output for sequential tests.  They take a lot of log lines usually. But yeah. We can keep it. This output is groupped per "test type" so it will be folded anyway and agree that it might be useful at times.

Reply:
Removed this variable altogether.